libsolidity/formal/SMTEncoder.cpp

```diff
@@ -444,6 +444,9 @@ void SMTEncoder::endVisit(Assignment const& _assignment)
 
 void SMTEncoder::endVisit(TupleExpression const& _tuple)
 {
+	if (_tuple.annotation().type->category() == Type::Category::Function)
+		return;
```

Comment:
This prevents SMTChecker from emitting warning "Internal error: Expression undefined for SMT solver".
This would happen if there is a function identifier nested in 1-tuple. Since we are not creating SMT representation for function identifiers (see `SMTEncoder::visit(MemberAccess const&)`), we should ignore these expressions here as well.